### DISCUSSION TITLE
blinking cursor

### DISCUSSION THREAD

**Discussion by pbwn** [qutebrowser/qutebrowser#6897] [General] [answered]
i'm using ubuntu 18.04. Qutebrowser ignores my system setting where the blink rate is set to 0. Is it possible to turn off the blink in the config file?

**Comment by toofar** [comment, marked as answer]
There is currently no setting for that. How does the falkon browser behave in this case? If they pick up the system setting maybe there is some way for us to. Otherwise maybe you can change it with a custom stylesheet?

**Comment by pbwn** [comment]
thanks. It's interesting: chromium blinks, epiphany doesnt, brave blinks, firefox doesnt etc. I am one of those people who cannot use anything that has a blinking cursor, i will use vieb for the time being

**Reply by The-Compiler** [reply]
Brave and QtWebEngine/qutebrowser are both based on Chromium, so that seems to be the culprit.
